script/Arcueid/Day 5/05_00_ARC05INTERLUDE.txt

```diff
@@ -201,6 +201,7 @@ He marches with the Dead in tow.
 [sha:8f6fe43ba488f5242a4b6585ec251c41d0b1fed5]{
 -- Page 360, Offset 9522.
 -- 垂直に移動する箱を抜けて、光あふれる地上に這い出る。
+//does cab have a meaning im not getting here????
```

Comment:
could we add "elevator" before cab? i know it obviously _isn't_ a taxi but i can't help but think of one when i see it 😓

Reply:
i don't think so, but if that's the case then it might be even more reason to modify cab. i mean, if vlov doesn't know wtf an elevator is then he probably wouldnt know "cab" either

Reply:
Manolo's point is a good one. I don't really think cab is a very good translation either way for 「垂直に移動する箱」

It really ought to be literal: "vertical moving box" or something.

Reply:
"vertically moving box" seems like the best middle ground. if both of you are good with it i'll go ahead and edit it